tests/fixtures/fixtures.cpp

```diff
@@ -267,4 +276,30 @@ GenTestItems(uint64_t count, uint64_t max_length, uint64_t max_index) {
     return result;
 }
 
+template <typename T>
+static T*
+CopyVector(const std::vector<T>& vec) {
```

Comment:
minor: CopyVector -> ToArray ?

Reply:
Copy means alloc another memory, but ToArray may loss this meaning